projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -315,6 +320,22 @@ private function load_value_from_callback( $item, $callback, $default = '' ) {
 		return $default;
 	}
 
+	/**
+	 * Loads any extra data for a task, calling the 'extra_data_callback' callback if it exists,
+	 * or null if there is no callback or the callback returns an empty array.
```

Comment:
Reading this fully, it feels like we should break up the sentences.
```suggestion
	 * Loads any extra data for a task, calling the 'extra_data_callback' callback to get the data if the callback is defined.
	 * Returns null if there is no callback or the callback returns an empty array or a non-array.
```